backend/src/server/uiRoutes.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import { fileURLToPath } from 'node:url';
+
+import { importMetaUrl } from './importMetaUtils';
+
+export default express.Router().use(
+	express.static(
+		fileURLToPath(new URL('../../../frontend/dist', importMetaUrl())),
```

Comment:
```suggestion
		fileURLToPath(new URL('@src/frontend/dist', importMetaUrl())),
```
does this work?

Reply:
Nope, because `@src` would be relative to _backend_, and we are needing the frontend build.